README.md

```diff
@@ -1 +1,34 @@
-# geekjob
\ No newline at end of file
+# GeekJob - Yet Another Job Board  
+
+:muscle::muscle::muscle: We are a small engineers group who are living & working in Berlin  
+
+:mask::mask: Observing the instability of market in these days due to the covid chaos we believe our geekJob channel will help you:  
+
+:fire: receive of the great (in our opinion) job updates which will be posted by our admins. but feel free to send us if you have any (we will post them on your behalf). sooner you will able to post by yourself  
+
+:fire: discuss on the specific job which we posted and do the advice if you need any on your career path. More than that we re willing to hear & support your current technical issues  
+
+## Job Posting Rules  
+
+:point_right: :point_right: Please read the posting rule [HERE!](POSTING-RULES.md) before submit PR.  
+
+## Quick Links
+
+:point_right: [DevOps Jobs](DevOps/README.md)  
+:point_right: [QA Jobs](QA-Engineer/README.md)  
+:point_right: [Software Jobs, BackEnd](Software-Engineer-BackEnd/README.md)  
+:point_right: [Software Jobs, FrontEnd](Software-Engineer-FrontEnd/README.md)  
+:point_right: [Software Jobs, IOS](Software-Engineer-IOS/README.md)  
+:point_right: [Software Jobs, Android](Software-Engineer-Android/README.md)  
+:point_right: [Software Jobs, ReactNative](Software-Engineer-ReactNative/README.md)  
```

Comment:
Should we have ReactNative as a dedicated category rather than apart of front-ends/mobile ones?

Reply:
I dunno. People use ReactNative to build a complete mobile app, similar to Android/IOS, so i thought it should be on a dedicated category. 

@vn-patriot any thought on this ?

Reply:
In my opinion, both are correct & insightful enough however I would make it a bit more simpler. So let say we create  2 categories 
- FrontEnd (Web)
- ReactNative (Mobile)

Delete Software Engineer IOS & Software Engineer Android 

However inside React Native mobile we can set some labels like (iOS) / (Android).

Reply:
Not all people use ReactNative for mobile development, I think it is better to just keep them IOS and Android.
My initial thought was React may dead in the future :-))

Reply:
ok so let's remove this category

Reply:
removed by commit 3c52e5a550cf7dcd6fe3095dd3257e6bcc3632e0